literals/prod.json

```diff
@@ -0,0 +1,10 @@
+{
+  "fishing_hours_description": "Total fishing hours for all vessels on the selected time period are determined by the Global Fishing Watch fishing detection algorithm. Our fishing algorithm identifies each AIS broadcast as 'likely fishing' or 'not fishing.' We then assign an amount of time to each AIS point calculated from the time between each AIS broadcast. Because data from AIS is not evenly spaced in time,  the amount of time assigned to each AIS point varies.",
```

Comment:
@j8seangel and @nerik I've created a ticket to merge the to literals.json files the one from dev and prod. The only difference is the domain and we already have that in `REACT_APP_SITE_URL`. https://github.com/GlobalFishingWatch/GFW-Tasks/issues/1154